junebug/channel.py

```diff
@@ -43,13 +43,13 @@ class Channel(object):
     APPLICATION_CLS_NAME = 'junebug.workers.MessageForwardingWorker'
 
     def __init__(
-            self, redis_manager, amqp_config, properties, id=None):
+            self, redis_manager, amqp_config, properties, ttl, id=None):
```

Comment:
This `__init__` is also getting pretty long, wondering if we should also pass it the/a config too?